platform/stm32f0xx/ldscripts/bootloader/memory_spaces.ld

```diff
@@ -0,0 +1,39 @@
+/*
+Linker subscript for memory definitions for the bootloader on STM32F072 with 128K Flash and 16K RAM.
+See https://uwmidsun.atlassian.net/l/c/mHxmqjvn for more information.
+*/
+
+ENTRY(Reset_Handler)
+
+/* bootloader size - this must be as small as possible while aligned on 2K and large enough */
+_bootloader_size = 16K;
+
+_flash_start = 0x08000000;
+_flash_size = 128K;
+_flash_page_size = 2K;
+
+_ram_start = 0x20000000;
+_ram_size = 16K;
+
+_bootloader_start = _flash_start;
+_config_page1_start = _bootloader_start + _bootloader_size;
```

Comment:
What are these config pages? I'm looking at the memory map but not sure I see it?
![image](https://user-images.githubusercontent.com/37023871/108633133-a03ccc00-7440-11eb-95e2-10651cb32215.png)

Reply:
https://www.st.com/resource/en/datasheet/stm32f072c8.pdf

Reply:
These are designations for the flash pages where we'll store the bootloader config - see https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/2030895112/Bootloader+Flash+over+CAN#Memory-Layout-and-Config. It's not an intrinsic part of the stm32 memory layout, just a section of flash I set aside so we don't overwrite it with the bootloader code or other such silliness.